docs/SRS/SRS.tex

```diff
@@ -749,7 +801,7 @@ \subsection{Effects on the Installed Systems}
 
 \subsection{Potential User Problems}
 \begin{itemize}
-    \item Accessibility to the command terminal requires authentication to be granted through the OpenID Connect Protocol. However, this is an industry-standard protocol for authentication and authorization, making it highly unlikely to encounter this problem.
+    \item Accessibility to the command terminal requires authentication to be granted through the Auth0 Protocol. However, this is an industry-standard protocol for authentication and authorization, making it highly unlikely to encounter this problem.
```

Comment:
Similarly here since OpenID Connect is the protocol and auth0 is the service:

Accessibility to the command terminal requires authentication to be granted through the OpenID Connect Protocol offered by Auth0. However, this is an industry-standard protocol for authentication and authorization, making it highly unlikely to encounter this problem.